src/Datagrid/ProxyQuery.php

```diff
@@ -202,6 +202,7 @@ public function execute(array $params = [], $hydrationMode = null)
 
         $query = $this->getDoctrineQuery();
```

Comment:
hm now we execute `->getDoctrineQuery()` twice? I think we can avoid that, or?

Reply:
All the logic could be in `if (\func_num_args() > 0) {`